### PR TITLE
DROP INDEX doc

### LINKED ISSUE #258: DROP INDEX statement
Complete reference documentation for the `DROP INDEX` statement: drop-index.md

Intro, Synopsis, and Required Privileges are complete. We need to complete **Parameters** and **Examples**. For style reference, see those sections in [`INSERT`](insert.html) or [`DELETE`](delete.html).

More reference:
- https://github.com/cockroachdb/cockroach/blob/master/sql/drop.go
- https://github.com/cockroachdb/cockroach/blob/master/sql/testdata/drop_index
- PostgreSQL docs: http://www.postgresql.org/docs/9.5/static/sql-dropindex.html
- MySQL docs: http://dev.mysql.com/doc/refman/5.7/en/drop-index.html

### PR DESCRIPTION
**Pull request by sploiselle**
Closes #536 
Closes #258

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/540)

### CODE REVIEW

File: drop-index.md [hunk 1]
```diff
@@ -14,12 +14,36 @@ The `DROP INDEX` [statement](sql-statements.html) removes an index from a table.
 
 ## Required Privileges
 
-The user must have the `CREATE` [privilege](privileges.html) on the table. 
+The user must have the `CREATE` [privilege](privileges.html) on each specified table.
+
```
Comment (tamird) [inline drop-index.md]: Spurious newline?

File: drop-index.md [hunk 2]
```diff
@@ -14,12 +14,36 @@ The `DROP INDEX` [statement](sql-statements.html) removes an index from a table.
...
 ## Parameters
 
 | Parameter | Description |
 |-----------|-------------|
-|  |  |
-
-
+| `IF EXISTS`	| Drop the named indexes if they exist; if they do not exist, do not return an error.|
+| `table_name`	| The name of the table with the index you want to drop. Find table names with [`SHOW TABLES`](show-tables.html).|
+| `index_name`	| The name of the index you want to drop. Find index names with [`SHOW INDEX`](show-index.html).<br/><br/>You cannot drop a table's `primary` index.|
```
Comment (tamird) [inline drop-index.md]: Throughout: s/index/indexes/g, s/table/tables/g

**Comment by tamird** [comment]
LGTM modulo comments. Please squash before merging; a commit message of the form "extension of the work by ..." is of little use to future readers.

File: drop-index.md (hunk 1 above)
Comment (sploiselle) [inline drop-index.md]: Spurious newline.

File: drop-index.md (hunk 2 above)
Comment (sploiselle) [inline drop-index.md]: Some elements apply only to one table/index. Others can apply to multiple, so they're pluralized. Sincerely, though, appreciate the attention to detail.

**Comment by sploiselle** [comment]
@tamird Appreciate the review and best practices for commits. Cleaned everything up.

**Pull request merged**